### ISSUE TITLE
Disable repeat when playing an entire album/artist

### ISSUE THREAD

**Issue by Martysh12** [OxygenCobalt/Auxio#847]
### Description

Disable repeat when pressing the "play" button on the album/artist screen.

### Problem solved

Whenever I press the "play" button, I intend to play the entirety of the album/artist, not just the first track over and over again (for that I'd simply select the first track). Currently, pressing "play" doesn't disable repeat:

https://github.com/user-attachments/assets/447d6c13-90be-4189-954b-fa514678cfcd

This issue creates some funny situations where I press the "play" button, put my phone away, cover my hands in cooking oil, and realise that I'm stuck on the first track of the album without being able to do anything.

### Other implementations

Gramophone seems to disable repeat when pressing the play icon on an artist/album.

### Benefit

It just makes more sense to have the "play" button play the entire album, instead of keeping the repeat state.

### Duplicates

- [X] I have checked this feature request for duplicates.
- [X] I have checked that this feature is not implemented in the [lastest version](https://github.com/OxygenCobalt/Auxio/releases).
- [X] I have checked the [Why Are These Features Missing?](https://github.com/OxygenCobalt/Auxio/wiki/Why-Are-These-Features-Missing%3F) page.
- [X] I agree to the [Contribution Guidelines](https://github.com/OxygenCobalt/Auxio/blob/dev/.github/CONTRIBUTING.md).

**Comment by OxygenCobalt** [comment]
This sounds like a good companion setting in the same vain as 'Remember Shuffle'. Will spec it out.

**Comment by OxygenCobalt** [comment]
So, I can spec this in three ways @Martysh12:
- Spec it exactly like you want, making the behavior to drop the repeat mode between playing from distinct sources. This is inconsistent with remember shuffle though.
- Spec it more broadly, making the behavior to drop the repeat mode when playing any new song (even from the same source, i.e if you're browsing around an artist).
- Make both Remember Shuffle and Remember Repeat both a type of dropdown with the prior options available at once (On, When playing from the same source, Never). This is really hard to explain coherently though.

**Comment by Martysh12** [comment]
@OxygenCobalt Could you elaborate on the 1st and the 3rd options please? What's a distinct source?

**Comment by OxygenCobalt** [comment]
A distinct source is like playing from an Album vs. an Artist. So if I went with Option 1, the repeat mode would only clear when switching playback to an Artist from an Album, rather than when playing a new song within the album @Martysh12.

It's hard to explain, hence why I am hesitant. I would rather do Option 2 since it's much easier to explain to users.

**Comment by Martysh12** [comment]
Option 2 sounds like it would clear Repeat One whenever I switch to a new track, so I'm not quite sure about it. Option 1 makes more sense to me.

**Comment by OxygenCobalt** [comment]
Yeah, that's what I want, but I can't figure out how to explain it clearly @Martysh12.

**Comment by Martysh12** [comment]
Hmm, maybe have a label next to the option with something like
"Clear repeat when switching to a different album/artist"?

@OxygenCobalt

**Comment by Martysh12** [comment]
![image](https://github.com/user-attachments/assets/915c0372-9203-45a2-8257-ffc72f2f0f3e)

Here, @OxygenCobalt, I made a mockup of the "clear repeat" option in settings.
Tbh, I think it makes a lot of sense UX-wise.

**Comment by OxygenCobalt** [comment]
Okay, this makes sense @Martysh12. For consistency, I might go and invert Remember shuffle (and remember pause) in the same way.
